challengers/Joyal-Poudel/hard/build-image-gallery/styles.css

```
@@ -1,242 +1,190 @@
-    to {
-        transform: translateY(0);
-        opacity: 1;
-    }
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
 }
 
-.lightbox-content img {
+.lb-content img {
```

Stylelint (17.6.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)